### PR TITLE
Vet clinic database: add "join table" for visits

### PR DESCRIPTION
**Pull request by yaredtekle022**
**These list of activities were done**
### schema.sql

>  Created a table named vets

> Created a "join table" called specializations

> Created a "join table" called visits
### data.sql

> Insert data for vets

>  Inserted data for specializations

> Inserted data for visits:

### Queries

- Who was the last animal seen by William Tatcher?
Screenshot 2023-08-31 at 23 00 08

-  How many different animals did Stephanie Mendez see?
Screenshot 2023-09-02 at 08 22 46

- List all vets and their specialties, including vets with no specialties.

Screenshot 2023-09-02 at 08 23 44

- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
Screenshot 2023-09-02 at 08 25 20

- What animal has the most visits to vets?
Screenshot 2023-09-02 at 08 29 30

- Who was Maisy Smith's first visit?

Screenshot 2023-09-02 at 08 27 37

- Details for most recent visit: animal information, vet information, and date of visit.
Screenshot 2023-09-02 at 08 30 50

- How many visits were with a vet that did not specialize in that animal's species?
Screenshot 2023-09-02 at 08 32 05

- What specialty should Maisy Smith consider getting? Look for the species she gets the most

Screenshot 2023-09-02 at 08 33 09

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -53,3 +53,46 @@ UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer
...
+    (7, 3, '2019-09-29');
+    (8, 4, '2020-10-03'),
+    (8, 4, '2020-11-04');
+    (9, 2, '2019-01-24'),
+    (9, 2, '2019-05-15'),
+    (9, 2, '2020-02-27'),
+    (9, 2, '2020-08-03'),
+    (10,3, '2020-05-24'),
+    (10,1, '2021-01-11');
+
```
Comment (t-yanick) [inline data.sql:100]: - [ ] Great job working on this milestone. Kindly check the inserted data. You seem to be missing 2 records. As per project requirements, there are 20 records.

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @yaredtekle022 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  All SQL commands put in the correct files
:heavy_check_mark: Query results screenshots added to PR body
:heavy_check_mark: Use of correct Gitflow


There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (usorfaitheloho) [reply data.sql:100]: As suggested by the previous reviewer, you should add the 2 missing visit data., I think the missing ones are the first two visits ⬇️ 

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
## Hi @yaredtekle022   :clap:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## To Highlight :clap:  
:arrow_forward: Git/Github flow best practices has been followed :heavy_check_mark: 
:arrow_forward: Good code quality :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — APPROVED** [review]
Hi @yaredtekle022 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Clean queries, correct data and meaningful schema

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**